dynatrace-aws-logs.sh

```diff
@@ -109,6 +108,32 @@ arguments:
     fi
   }
 
+  function check_log_ingest_url() {
+  if RESPONSE=$(curl -k -s -X POST -d "$(generate_test_log)" "$TARGET_URL/api/v2/logs/ingest" -w "<<HTTP_CODE>>%{http_code}" -H "accept: application/json; charset=utf-8" -H "Content-Type: application/json; charset=utf-8" -H "Authorization: Api-Token $TARGET_API_TOKEN"); then
```

Comment:
Are there any timeouts? This might hang up for a while if there is some firewall etc.

Reply:
Yes, there is timeout 20 s, I've added it explicitly to curl